statsmodels/stats/meta_analysis.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Apr  2 14:34:25 2020
+
+Author: Josef Perktold
+License: BSD-3
+
+"""
+
+import numpy as np
+import pandas as pd
+from scipy import stats
+
+
+class CombineResults(object):
+
+    def __init__(self, **kwds):
+        self.__dict__.update(kwds)
+        self._ini_keys = list(kwds.keys())
+
+        # explained variance measures
+        self.h2 = self.q / (self.k - 1)
+        self.i2 = 1 - 1 / self.h2
+
+    def test_heterogeneity(self):
```

Comment:
DerSimonian and Laird 1986 talk about `test_homogeneity`
null is homogeneity, i.e. equal means

Reply:
renamed